example/provider-local/seed-kind/local-ipv6/seed.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: core.gardener.cloud/v1beta1
```

Comment:
Would it be possible to work with patches to prevent copying the entire manifest? Like https://github.com/gardener/gardener/tree/master/example/provider-local/seed-kind2/local?